src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -94,14 +99,18 @@ export const MultiSelect: FC<MultiSelectProps> = ({
                 <div
                     {...mergeProps(buttonProps, focusProps)}
                     ref={triggerRef}
-                    className="tw-flex-1 tw-flex tw-flex-wrap tw-gap-1 tw-pl-[19px] tw-py-[11px] tw-min-h-[34px] tw-outline-none tw-pr-7"
+                    className={merge([
+                        'tw-flex-1 tw-flex tw-flex-wrap tw-gap-1 tw-pl-[19px] tw-min-h-[34px] tw-outline-none tw-pr-7',
```

Comment:
Those custom values feel strange for the padding. Is there a link to the figma file?

Reply:
I was more or less trying to get values from this one:
https://www.figma.com/file/WfEbqZI0nnPWHN43MCtvnm/Fondue---Inputs?node-id=83%3A22094

This is more like a middle ground, we need a multi-select input that is in line with other inputs (in terms of height).

Reply:
I see but could it be adjusted for the other inputs? Fondue Inputs have a size of 36px and I would rather have some consistency in the design system to have similar sizing. I would rather get similar values from the Text Input Component: https://fondue-components.frontify.com/?path=/story/components-text-input--text
But that's a design decision. 🙈 It would be better to have Designer feedback on this one.

On the other hand, specifying the height for the Small variant creates an issue when we select many options.
The tags overlap, and the multi-select should expand so I would use min-height instead.
![Screenshot 2022-08-03 at 11 22 53](https://user-images.githubusercontent.com/35405438/182574594-18bdcf00-de74-475c-be3a-cc866a242e8b.png)

Also, the minimum height of the default size of MultiSelect as shown in Figma is 38px: https://www.figma.com/file/M0dX8EsqEgnHxNStt6Lgud/Fondue-Components---WIP?node-id=12222%3A190439

I would use something like this:

```
className={merge([
    'tw-flex-1 tw-flex tw-flex-wrap tw-gap-1 tw-pl-[19px] tw-outline-none tw-pr-7',
     size === MultiSelectSize.Small ? 'tw-min-h-9 tw-items-center' : 'tw-min-h-[38px] tw-py-[11px]',
])}
```
What do you think?

Reply:
That is a very good point, good catch. 36px with borders is 34px without, but nevertheless I changed the approach to use paddings in all cases to avoid scenarios like your screenshot. This way it is more consistent, what do you think? (I created a helpers file for this similar to what you have for Badges)

Reply:
Yes, makes sense to me! Let's keep it to 34px so it is 36px with the border. Even if the Figma says 38px, I would rather have it exactly like the Text Input component. But I'll ask Jamie when he's back anyway.